resources/views/exhibitions/index.blade.php

```diff
@@ -57,13 +57,16 @@
     </div>
 @endsection
 @if(!empty($future['data'] ))
+    @php
```

Comment:
You don't need this - change the controller (https://github.com/FitzwilliamMuseum/fitz-main-website/blob/master/app/Http/Controllers/exhibitionsController.php#L57) parameters $future = Exhibitions::listFuture('future','exhibition_start_date');
Sort is either -exhibition_start_date or exhibition_start_date depending on order you want.

Reply:
Thanks Ant! I've changed it to what you suggested and it worked! This PR should be updated with that change now.

Reply:
You're welcome, Dan :)